src/arrayMethodSort.js

```diff
@@ -4,9 +4,25 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
```

Comment:
The default compare function should convert the elements to strings and compare their sequences of UTF-16 code units values. Using template literals is not necessary and can be simplified to String(a) and String(b).